go.mod

```diff
@@ -9,9 +11,9 @@ require (
 	github.com/google/gofuzz v1.2.0
 	github.com/google/uuid v1.6.0
 	github.com/gophercloud/gophercloud/v2 v2.4.0
-	github.com/gophercloud/utils/v2 v2.0.0-20241008104625-7cbb8fd76bb7
+	github.com/gophercloud/utils/v2 v2.0.0-20241209100706-e3a3b7c07d26
```

Comment:
I wonder why this was bumped. ORC shouldn't be exporting anything gophercloud related 🤔

Reply:
I don't know TBH, this is generated by `go mod tidy`.